### PR TITLE
Change indent size from 2 spaces to 4 spaces

### PR DESCRIPTION
**Pull request by rougsig**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,6 @@
 .gradle
 /local.properties
 /.idea
-!/.idea/codeStyles
```
Comment (LionZXY) [inline .gitignore:5]: Return codestyle, but set 4 space

**Review by LionZXY — CHANGES_REQUESTED** [review]

File: core/src/test/java/com/flipper/core/ExampleUnitTest.kt [hunk 2]
```diff
@@ -9,8 +9,8 @@ import org.junit.Test
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class ExampleUnitTest {
-  @Test
-  fun addition_isCorrect() {
-    assertEquals(4, 2 + 2)
-  }
+    @Test
+    fun addition_isCorrect() {
```
Comment (LionZXY) [inline core/src/test/java/com/flipper/core/ExampleUnitTest.kt:13]: remove test, please

**Review by LionZXY — CHANGES_REQUESTED** [review]

File: core/src/test/java/com/flipper/core/ExampleUnitTest.kt (hunk 2 above)
Comment (rougsig) [reply core/src/test/java/com/flipper/core/ExampleUnitTest.kt:13]: Ok. In next PR

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -19,19 +19,7 @@
       </option>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
-    <codeStyleSettings language="JAVA">
-      <indentOptions>
-        <option name="INDENT_SIZE" value="2" />
```
Comment (LionZXY) [inline .idea/codeStyles/Project.xml:24]: Can you change it to 4?

**Review by LionZXY — CHANGES_REQUESTED** [review]

**Review by LionZXY — APPROVED** [review]

**Pull request merged**